BlackRevival.Common/GameDB/Tutorial/TutorialTransmitData.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+using BlackRevival.Common.Enums;
+using BlackRevival.Common.Model;
+
+namespace BlackRevival.Common.GameDB.Tutorial;
+
+public class TutorialTransmit
+{
+	[JsonPropertyName("cod")]
```

Comment:
These all have to have getters/setters or system.text.json doesn't pick them up when Serializing/Deserializing